app/src/main/java/com/lagradost/cloudstream3/MainActivity.kt

```diff
@@ -1174,8 +1178,12 @@ class MainActivity : AppCompatActivity(), ColorPickerDialogListener {
             null
         }
 
-        changeStatusBarState(isEmulatorSettings())
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M && !isAppOptimised && !isTrueTvSettings()
+            && !isTvSettings() && !isEmulatorSettings()) {
+            showBatteryOptimizationDialog()
```

Comment:
This also has no settings related option to access this after doing it for the first time